### PR TITLE
K8s improvements summer 2021

### PR DESCRIPTION
**Pull request by godber**
This PR will address the issues in the following milestone:

https://github.com/terascope/teraslice/milestone/8

### CODE REVIEW

**Comment by godber** [comment]
Issue #2938 is the first one I've addressed.

**Comment by godber** [comment]
Manually checking the `podAntiAffinity` on a pod on a running test job on a cluster where I set `    kubernetes_worker_antiaffinity: true`, I get the following:

```
spec:
  affinity:
    podAntiAffinity:
      preferredDuringSchedulingIgnoredDuringExecution:
      - podAffinityTerm:
          labelSelector:
            matchExpressions:
            - key: app.kubernetes.io/name
              operator: In
              values:
              - teraslice
            - key: app.kubernetes.io/instance
              operator: In
              values:
              - ts-dev1
          topologyKey: kubernetes.io/hostname
        weight: 1
```

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM terascope/node-base:14.17.0
+FROM terascope/node-base:14.17.5
```
Comment (peterdemartini) [inline Dockerfile:1]: I think there is a `14.17.6` now.

```suggestion
FROM terascope/node-base:14.17.6
```

**Review by peterdemartini — CHANGES_REQUESTED** [review]
One change recommended, and also teraslice needs to be bumped

**Review by peterdemartini — APPROVED** [review]
LGTM

**Pull request merged**